ui/job-view/details/tabs/PerformanceTab.jsx

```diff
@@ -16,6 +86,63 @@ export default class PerformanceTab extends React.PureComponent {
         role="region"
         aria-label="Performance"
       >
+        <div className="performance-panel-actions">
+          {
+            // If there is a profiler link, show this first. This is most likely
+            // the primary action of the user here.
+            profilerLink
+          }
+          {
+            // Just to be safe, use the same isPerfTest check the other
+            // "Create Gecko Profile" button uses in the action menu.
+            isPerfTest(selectedJobFull) ? (
+              <button
+                type="button"
+                className={`btn btn-${
+                  // Only make this primary if there is no profiler link.
+                  profilerLink ? 'secondary' : 'primary'
+                } btn-sm`}
+                onClick={this.createGeckoProfile}
+                title={
+                  'Trigger another run of this test with the profiler enabled. The ' +
+                  'profile can then be viewed in the Firefox Profiler.'
+                }
+              >
+                <FontAwesomeIcon icon={faRedo} className="mr-2" />
+                {profilerLink
+                  ? 'Re-trigger performance profile'
+                  : 'Generate performance profile'}
+              </button>
+            ) : null
+          }
+          <a
+            href={getCompareChooserUrl({
+              newProject: repoName,
+              newRevision: revision,
+            })}
+            target="_blank"
+            rel="noopener noreferrer"
+            className="btn btn-secondary btn-sm"
+          >
+            <FontAwesomeIcon icon={faTable} className="mr-2" />
+            Compare against another revision
+          </a>
+        </div>
+        {
+          // It can be confusing after triggering a profile what happens next. The
+          // job list only gets populated later. This notification will help the
+          // user know the next action.
+          triggeredGeckoProfiles > 0 ? (
+            <div className="alert alert-info m-1" role="alert">
+              <FontAwesomeIcon icon={faInfoCircle} className="mr-1" />
+              {triggeredGeckoProfiles === 1
+                ? `Triggered ${triggeredGeckoProfiles} profiler run. It will show up ` +
```

Comment:
This is useful, yet we could further improve it. Either by mentioning the `-p` suffix or by providing the full symbol name for the profiled perf job that we expect (e.g. T(damp-p)).

Reply:
I don't think we have anything that will give us the full name here, which is why I kept it generic. This works for more than just damp tests.

<img width="936" alt="image" src="https://user-images.githubusercontent.com/1588648/95496586-aab16a00-0966-11eb-8d27-ff9e3c10d215.png">

If there is a way to get this information I think it would be nice to display, but I would prefer to follow up.